Automattic/jetpack
# My Jetpack: Add tracking events to Plans section external links

Fixes #23168

#### Changes proposed in this Pull Request:

* Added tracking events to Plans section external links

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


Yes, it adds two new links to the set of clicks we are tracking in the My Jetpack page. Under the Plans section, there's always a link to either "Purchase a plan" (when there's no current plan) or a link to "Manage your plan". We are now tracking clicks in these links.

#### Testing instructions:






* Go to My Jetpack
* enable debugging of tracking events (type: `localStorage.setItem('debug', 'dops:analytics');` in the console)
* Click "Manage your plan" or "Purchase a plan" and verify the correspondent event is tracked
* Make sure the event tracked matches the link label and destination
* Are the event names good?
* Check for typos
